IbisITK/itkVolumeReconstructionOpenCL/itkGPUVolumeReconstruction.hxx

```diff
@@ -667,22 +673,29 @@ GPUVolumeReconstruction< TImage >
     {
       ImagePixelType reconstructedValue = weightedValue / weight;
       m_ReconstructedVolume->SetPixel(m_ReconstructedVolume->ComputeIndex(i), reconstructedValue);
+#ifdef DEBUG
+      m++;
+#endif
     }
     else
     {
       m_ReconstructedVolume->SetPixel(m_ReconstructedVolume->ComputeIndex(i), 0.0);
+#ifdef DEBUG
+      n++;
+#endif
     }
   }
   clockSettingValue.Stop();
 
 #ifdef DEBUG
+    std::cout << "nbrOfPixelsInVolume = " << nbrOfPixelsInVolume << " m = " << m  << " n = " << n << std::endl;
     std::cerr << "Time to Pixel Values with for loop:\t" << clockSettingValue.GetMean() << std::endl;
 #endif    
 
 #ifdef DEBUG
```

Comment:
This DEBUG macro, when is it enabled? We might need a more specific name to make sure it is not ON whenever with build a debug version on some platform.